docs/index.bs

```diff
@@ -163,13 +163,17 @@ spec: webappsec-referrer-policy; urlPrefix: https://w3c.github.io/webappsec-refe
 
     A [=/service worker=] has an associated <dfn export id="dfn-script-resource-map">script resource map</dfn> which is an <a>ordered map</a> where the keys are [=/URLs=] and the values are [=/responses=].
 
+    A [=/service worker=] has an associated  <dfn export id="dfn-set-of-used-scripts">set of used scripts</dfn> (a [=ordered set|set=]) whose [=list/item=] is a [=/URL=]. It is initially empty.
```

Comment:
I think "It is initially a new [=ordered set|set=]" is the correct language here (and below).

Reply:
Thank you! I see the infra spec examples has instances of "Let result be an empty list." or "Let tokens be a list of strings, initially empty.". But I see it also has "Let list be a new list." OK... I guess we'll do "new set"... seems wordy though.

Reply:
Yeah, it makes more sense when something can be an ordered set or null. Eg:

> A thing has an associated **whatever** (a set or null). It is initially empty.

Since it can be a set or null, it isn't totally clear that "initially empty" means a new empty set.

Reply:
OK I see.